2-mandatory/3-groceries.js

```diff
@@ -26,14 +26,49 @@ Exercise 1:
   The weeklyGroceriesToBuy array shouldn't contain any repeating items.
 */
 // Gather all week item names into this array
+let weeklyGroceriesToBuy = Object.values(weeklyMealPlan)
```

Comment:
In light of the CYF guidelines regarding JavaScript, 
we need to get into the habit of using longer, descriptive variable names.

In your example using just _a and b_ does not help when trying to figure out what your routine is doing!
Hint: https://syllabus.codeyourfuture.io/guides/code-style-guide#think-of-good-names-for-your-variables

Reply:
Yes, you may name a to **tmpWeeklyGroceriesToBuy** (tmp short-form for temporally)